share/templates/rst/index.rst.j2

```diff
@@ -44,6 +44,10 @@
 {{ output.text | indent }}
 {% endblock stream %}
 
+{% block data_native %}
+{{ output.data['text/restructuredtext'] }}
```

Comment:
Should it also support `text/x-rst` and `text/prs.fallenstein.rst`? `text/restructuredtext` seems pretty non-standard to me (although I acknowledge that this is what nbconverte is already using for some reason)

Reply:
Huh, good point. :thinking: 

I think it's best for users to have 1 MIME type for rst, rather than trying to wrangle several. So maybe we should change `RSTExporter.output_mimetype` as well - probably to `text/x-rst`, since that's what [docutils suggests](https://docutils.sourceforge.io/FAQ.html#what-s-the-official-mime-type-for-restructuredtext-data).

What would break if we do that?

- Raw cells in notebooks marked with the mimetype `text/restructuredtext` would stop showing up - but we could work around that by overriding `raw_mimetypes` to add back `text/restructuredtext` for compatibility.
- There's a mechanism for finding templates based on `output_mimetype`, so custom templates marked `text/restructuredtext` would no longer work. I can't imagine that many people have custom rst templates, and it's an easy change, so I'd be inclined to document that as a breaking change rather than adding extra complexity to keep them working.

Does that sound reasonable?

Reply:
I think it is reasonable. If I were doing the change I would probably try to make it as backward compatible as possible.

Reply:
Feel free to ping me for review once you think it is ready

Reply:
I trust your judgement. If your assessment is accurate, I would say it might be ok to release and see if anyone complains and only then add back the extra compatibility layer.

Reply:
Thanks!